Dockerfile

```diff
@@ -1,20 +1,29 @@
+# Use a base image that supports GUI applications
 FROM jlesage/baseimage-gui:ubuntu-22.04-v4 AS build
 
+# Maintainer information
 MAINTAINER Bjoern Gruening, bjoern.gruening@gmail.com
 
 RUN apt-get update -y && \
-     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-         ca-certificates \
-         wget \
-         unzip \
-         libgl1 \
-         qt5dxcb-plugin && \
-     rm -rf /var/lib/apt/lists/*
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        ca-certificates \
+        wget \
+        unzip \
+        libgl1 \
+        qt5dxcb-plugin &&\
+    rm -rf /var/lib/apt/lists/*
+
+# Define working directory
```

Comment:
why do you need this here, you are overwriting it below

Reply:
so I don't know  why but if I remove it, it does not work

Reply:
Ok, dont' worry